packages/gateway-client/src/worker/index.ts

```diff
@@ -769,28 +796,41 @@ async function getBootstrapList(skipFetch = false) {
 
 function websocketAddressFilter(addresses: MultiaddrType[]) {
     const res = WSAllfilter(addresses).filter((addr: MultiaddrType) => {
-        // console.log('filter?', addr.toString());
+        // console.log('filter?', addr.toString(), selt.lat);
         return self.latencySet.has(addr.toString());
     });
     // console.log('ran filter', res);
     return res;
 }
 
+const getQuickestPath = (): string | null => {
+    let quickest = Infinity;
+    let quickestAddr = null;
+    for (const [addr, latency] of self.latencyMap.entries()) {
+        if (latency < quickest) {
+            quickest = latency;
+            quickestAddr = addr;
+        }
+    }
+    return quickestAddr;
+};
+
 async function main() {
     // self.window.localStorage.debug = (await localforage.getItem('debug')) || ""
 
-    const bootstraplist = await getBootstrapList();
-    initCheckAddresses(bootstraplist);
+    const bootstraplist = await initCheckAddresses(await getBootstrapList());
+    console.log('bootstraplist', bootstraplist);
+    // await initCheckAddresses(bootstraplist);
     // update status
     self.status.serverPeer = ServerPeerStatus.BOOTSTRAPPED;
 
     self.status.relays.push(...bootstraplist.slice(2));
 
-    const datastore = new LevelDatastore('./libp2p');
-    await datastore.open(); // level database must be ready before node boot
+    // const datastore = new LevelDatastore('./libp2p');
+    // await datastore.open(); // level database must be ready before node boot
     const serverID = bootstraplist[0].split('/').pop();
     const node = await createLibp2p({
-        datastore,
+        // datastore,
```

Comment:
Are we using some sort of default datastore now?

Reply:
in-memory, will get reset every time the script restarts, but we're explicitely keeping track of our own relays.